core-sitemaps.php

```diff
@@ -18,58 +18,58 @@
  * Description:       A feature plugin to integrate basic XML Sitemaps in WordPress Core
  * Author:            Core Sitemaps Plugin Contributors
  * Author URI:        https://github.com/GoogleChromeLabs/wp-sitemaps/graphs/contributors
- * Text Domain:       core-sitemaps
+ * Text Domain:       sitemaps
```

Comment:
We cannot change the text domain, it must match the plugin slug from https://wordpress.org/plugins/core-sitemaps/

Reply:
👍 right! We will remove the text domain for core.

Reply:
reverted text domain changes in https://github.com/GoogleChromeLabs/wp-sitemaps/pull/182/commits/76d8240f06999f1c7d2a2439ec4bf995204c3cef